package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "concord",
-  "version": "0.0.5",
+  "version": "0.1.0",
```

Comment:
No reason for not remaining in `0.0.6` unless we are more confident in `0.1.0` than `0.0.5` to be a release candidate.

Reply:
I'd like to go up to `0.1.0` so we can use `^0.1.0` as a dependency and get patch updates.